public/test/mock-api/worker.ts

```diff
@@ -0,0 +1,5 @@
+import { setupWorker } from 'msw/browser';
+
+import allHandlers from 'app/features/alerting/unified/mocks/server/all-handlers';
```

Comment:
Alerting uses MSW quite a lot throughout tests - we export all of our handlers as we already set up an MSW server for running tests. This could be generalised as we go, and for now I think only Alerting has a structured/significant enough MSW setup to leverage this right now, but it can easily be extended